### PR TITLE
SignUp-Step 1

### PR DESCRIPTION
**Pull request by ppmarkek**

### CODE REVIEW

File: src/components/atoms/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -3,27 +3,29 @@ import { CheckboxGrid, StyledCheckbox } from './style';
...
+    return (
         <CheckboxGrid container>
             <StyledCheckbox>
-                <input type="checkbox" id="checkbox" onChange={(event) => handleChange(event)} />
-                <label htmlFor="checkbox" />
+                <input type='checkbox' id='checkbox' onChange={event => handleChange(event)} />
+                <label htmlFor='checkbox' />
             </StyledCheckbox>
-            <Text variant={'H5'} small={''} light={textColor}>{text}</Text>
+            <Text variant={'Light'} small={''} light={textColor}>
```
Comment (k0wapa) [inline src/components/atoms/Checkbox/Checkbox.tsx:24]: Variant should have typed variants and not string

File: src/components/pages/Login/style.ts [hunk 2]
```diff
@@ -3,77 +3,77 @@ import { Link } from 'react-router-dom';
...
 
 export const OrGrid = styled(Grid)`
-    align-items: center;
-    padding: 47px 0px;
-    gap: 16px;
-    width: unset;
+  align-items: center;
+  padding: 47px 0px;
+  gap: 16px;
+  width: unset;
```
Comment (k0wapa) [inline src/components/pages/Login/style.ts:36]: Why unset?

File: src/components/pages/SignUp/Step1/Step1.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import Google from '../../../../assets/Icon/google.svg';
+import Facebook from '../../../../assets/Icon/facebook.svg';
+import Twitter from '../../../../assets/Icon/twitter.svg';
+
+const Step1 = () => {
+    return (
+        <Wrapper container>
+            <Title container>
+                <Grid>
+                    <Text variant={'H1'} small={''} light={''}>
```
Comment (k0wapa) [inline src/components/pages/SignUp/Step1/Step1.tsx:16]: What's the purpose of light and small props?

File: src/components/pages/SignUp/Step1/Step1.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            Sign Up to getting started.
+                    </Text>
+                </Grid>
+
+                <Text variant={'Light'} small={''} light={''}>
+          Enter your details to proceed further
+                </Text>
+            </Title>
+            <Grid container gap={'36px'} flexDirection={'column'} width={'unset'}>
```
Comment (k0wapa) [inline src/components/pages/SignUp/Step1/Step1.tsx:28]: Gap as a string in brackets?

File: src/components/pages/SignUp/Step1/style.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  padding-bottom: 142px;
+`;
+
+export const Title = styled(Grid)`
+  width: 400px;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  gap: 11px;
+  padding-bottom: 59px;
```
Comment (k0wapa) [inline src/components/pages/SignUp/Step1/style.ts:19]: Round numbers everywhere

**Review by k0wapa — CHANGES_REQUESTED** [review]

File: src/components/pages/Login/style.ts (hunk 2 above)
Comment (ppmarkek) [reply src/components/pages/Login/style.ts:36]: Because I don't need 100% width

File: src/components/pages/SignUp/Step1/Step1.tsx (hunk 3 above)
Comment (ppmarkek) [reply src/components/pages/SignUp/Step1/Step1.tsx:16]: Light need to change a color and small need to change font-size

File: src/components/pages/SignUp/Step1/Step1.tsx (hunk 4 above)
Comment (ppmarkek) [reply src/components/pages/SignUp/Step1/Step1.tsx:28]: I fix this

**Pull request merged**